src/Scene/Node.js

```diff
@@ -21,7 +21,7 @@ function Node() {
     this.screenSpaceError = 0.0;
     this.loaded = false;
     this.pendingSubdivision = false;
```

Comment:
it's possible to remove `this.pendingSubdivision` and use `this.layerUpdateState = {};`?